index.html

```diff
@@ -91,6 +91,88 @@ <h2>Statement of Intent</h2>
 
 <section>
 <h2>Code</h2>
+
+	<h3 id="unacceptablebehaviour">
+        <strong>Unacceptable behaviour</strong>
+      </h3>
+
+      <p>Some behaviors run counter to the Code of Ethics and Professional Conduct. This list of unacceptable behaviors does not cover every case. Each person you interact with is unique, and behavior must be assessed on an individual level. Ensuring that your behavior does not have a negative impact is your responsibility.</p>
+
+      <ul>
+        <li>Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness,
+          neurotype, physical appearance, body, age, race, socio-economic status, ethnicity, nationality, language, or religion</li>
+
+        <li>Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health,
+          parenting, drugs, and employment</li>
+
+        <li>Deliberate misgendering or use of ‘dead’ or rejected names</li>
+
+        <li>Gratuitous or off-topic sexual images or behaviour in spaces where they are not appropriate</li>
+
+        <li>Physical contact and simulated physical contact (e.g., textual descriptions like “hug” or “backrub”) without consent
+          or after a request to stop</li>
+
+        <li>Threats of violence</li>
+
+        <li>Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm</li>
+
+        <li>Deliberate intimidation</li>
+
+        <li>Stalking or following</li>
+
+        <li>Harassing photography or recording, including logging online activity for harassment purposes</li>
+
+        <li>Sustained disruption of discussion</li>
+
+        <li>Unwelcome sexual attention</li>
+
+        <li>Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others</li>
+
+        <li>Continued one-on-one communication after requests to cease</li>
+
+        <li>Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect other
```

Comment:
In this case I would disagree, outing is definitely common enough parlance which is specifically relating to lgbtq people.

Reply:
OK, I was probably being old-fashioned. I accept this because you tell me, and also because it's listed as a definition (the 6th and final one for the word "outing") in my 26-year-old Shorter Oxford English Dictionary, and even if it was new then (it wasn't), quarter of a century is long enough for people to get with the times, even me.

That being the case, we can remove the quotation marks.

Reply:
Opened issue #44 to address reverse -isms and tone

Reply:
Thank you. I removed quotes.